themes/tiddlywiki/vanilla/ThemeTweaks.tid

```diff
@@ -2,6 +2,12 @@ title: $:/themes/tiddlywiki/vanilla/themetweaks
 tags: $:/tags/ControlPanel/Appearance
 caption: {{$:/language/ThemeTweaks/ThemeTweaks}}
 
+\function get.theme-tweaks.metric() [get.theme<metric>]
+
+\procedure get_theme-tweaks_lingo() <$transclude $variable="lingo" title={{{ [[Metrics/]addsuffix<metricName>] }}}/>
+
+\procedure get_theme-tweaks_lingo-hint() <$transclude $variable="lingo" title={{{ [[Metrics/]addsuffix<metricName>addsuffix[/Hint]] }}}/>
```

Comment:
We do not use underscores in the core at the moment and you should not start with it. It will cause consistency problems. So `get-theme-...` should be fine

If you think it is too similar to the function names, `getThemeTweaksLingo()` may be an option too